pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -292,18 +292,18 @@
             "#import": [ "ubuntu.14.04", "ubuntu-x64" ]
         },
 
-        "ubuntu.14.10": {
```

Comment:
We must have all versions.  This file defines every discrete identifier.  If CLI will use an identifier (or the identifier is needed for packaging) it must appear here.  Removing entries like this change does is breaking.

Imports should only be used if there is some binary compatibility guarantee between the releases.